AgentDiscoveries-Frontend/app/src/components/navbar.js

```diff
@@ -0,0 +1,53 @@
+import * as React from "react";
+import {
+    Navbar,
+    NavItem,
+    Nav,
+    NavDropdown,
+    MenuItem
+ } from "react-bootstrap";
+
+export default class Login extends React.Component {

```

Comment:
This component isn't being used, I expect we will end up using it when we implement routing